components/filter/filter.js

```diff
@@ -228,6 +231,15 @@ class Filter extends LocalizeCoreElement(RtlMixin(LitElement)) {
 
 		const dimension = singleDimension ? this._dimensions[0] : this._dimensions.find(dimension => dimension.key === this._activeDimensionKey);
 
+		const clear = html`
+			<d2l-button-subtle
```

Comment:
I felt it was easiest to just use a `d2l-button-subtle` here rather than a `d2l-selection-action` if it's a list dimension and a `d2l-button-subtle` if it's not.  I have the `dimension.appliedCount` readily available to do the same `require-selection` check, and I also want to disable the button if values are loading.